podcasts/OpmlImporter.swift

```diff
@@ -78,7 +78,7 @@ class OpmlImporter: Operation, XMLParserDelegate {
                     NotificationCenter.postOnMainThread(notification: Constants.Notifications.opmlImportCompleted)
                     return
```

Comment:
I think we need to remove this `return` or else if we are doing a smaller import the event bellow will not be tracked.